RPI/csproject_co2_reading.sql

```diff
@@ -0,0 +1,10 @@
+-- Sequence and defined type
+CREATE SEQUENCE IF NOT EXISTS csproject_co2_reading_id_seq;
+
+-- Table Definition
+CREATE TABLE "public"."csproject_co2_reading" (
```

Comment:
Currently I only use one table to record the reading. It's a basic table with id, value, and created_at.